src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -195,7 +194,7 @@ public Elevator() {
     // Simulation
     if (Robot.isSimulation()) {
       simElvMotor1 = elevatorMotor1.getSimState();
-      simElvMotor1.Orientation = ChassisReference.Clockwise_Positive;
+      simElvMotor1.Orientation = ChassisReference.CounterClockwise_Positive;
```

Comment:
This won't necessarily match the real orientation. May want to double check simulation to verify.